webpack.config.js

```diff
@@ -3,19 +3,19 @@ import TerserPlugin from "terser-webpack-plugin";
 import CopyPlugin from "copy-webpack-plugin";
 import path from "path";
 import * as url from "url";
-import glob from "glob";
+import { globSync } from "glob";
```

Comment:
@jessevanmuijden:
Change glob to globSync because of new glob dependency version.
Can you check the build, it cannot find the .ts files anymore (missing ./) in the path